setup.py

```diff
@@ -14,7 +14,7 @@ def run_tests(self):
         errno = pytest.main(self.test_args)
         sys.exit(errno)
 
-readme = open('README.txt').read()
```

Comment:
Please keep README.txt as is as this will create packaging / manifest problems downstream.